ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -772,7 +772,6 @@ tests:
     cluster_profile: equinix-ocp-metal
     env:
       DEVSCRIPTS_CONFIG: |
-        AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
```

Comment:
IINW partial override is not supported, the env var `DEVSCRIPTS_CONFIG` defined at the job level is going to replace fully the one defined at the workflow level. For such cases it would be probably better to use the generic workflow, which does not define any specific configuration

Reply:
Thanks, I definitely missed that. Should be fixed now.